go-controller/pkg/node/healthcheck.go

```diff
@@ -115,15 +122,23 @@ func checkForStaleOVSInterfaces(stopChan chan struct{}) {
 	}
 }
 
+type openflowManager struct {
```

Comment:
fyi I'll plan on refactoring this later to be more like the hybrid overlay openflow manager (probably just combine the 2 things into an openflow manager impl)